lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -122,6 +122,8 @@ def encoded_string(string)
         end
 
         def indent_lines(lines, failure_number)
+          return lines if failure_number.nil?
```

Comment:
This has been removed, I've decided it's probably better to keep the output similar to the console anyway.